crypto/sha/keccak1600.c

```diff
@@ -321,6 +343,149 @@ void KeccakF1600(uint64_t A[5][5])
     }
 }
 
+#elif defined(KECCAK_1X_ALT)
+/*
+ * This is variant of above KECCAK_1X that presses temporary storage
+ * even further, but at cost of more write references to A[5][5].
+ * It's less suitable if A[5][5] is memory bound, but better if it's
```

Comment:
> I don't know what "presses temporary storage" means. Uses more ?

No, less. 1X uses  C[5], D[5], T[2][5], while 1X_ALT - C[5], D[5]. And with latest commit C[5], D[2] is actually sufficient, but I left D[5] for better readability. Allegory for "presses" is if you want to squeeze you press. But given confusion of course it would be appropriate to re-word. "minimizes temporary storage even further"? "reduces requirement for temporary storage even further"? If there will be no explicit reply, I'll take the latter.

> By "A[5][5]" you mean the whole array, not just that one subscript?

It's direct reference to prototype below, which reads
```
static void Round(uint64_t A[5][5], size_t i)
```
But sure, one can replace it with A[][] (in commentary, not prototype)...